doc/users/config_file.rst

```diff
@@ -146,18 +150,26 @@ Execution
     crashfiles allow portability across machines and shorter load time.
     (possible values: ``pklz`` and ``txt``; default value: ``pklz``)
 
+*resource_monitor*
+    Enables monitoring the resources occupation (possible values: ``true`` and
+    ``false``; default value: ``false``)
+
+*resource_monitor_frequency*
+    Sampling period (in seconds) between measurements of resources (memory, cpus)
+    being used by an interface. Requires ``resource_monitor`` to be ``true``.
```

Comment:
is there a default when `resource_monitor=='true'`?

Reply:
added default value to the documentation